pyTigerGraph/gds/dataloaders.py

```diff
@@ -1514,6 +1514,23 @@ def __init__(
         # Install query
         self.query_name = self._install_query()
 
+    def metadata(self, additional_v_types=None, additional_e_types=None) -> Tuple[list, list]:
```

Comment:
Looks like this function is the same for all loaders. Why not putting it in the baseloader?

Reply:
It was because it doesn't apply to vertex, edge, or nodepiece loaders (at least not directly I guess). Can definitely put it in the baseloader though.